src/utils/dev.ts

```diff
@@ -0,0 +1,3 @@
+export const isDev: boolean = process.env.NODE_ENV === 'development'
+export const isTest: boolean = process.env.NODE_ENV === 'test'
+export const isNotProd: boolean = process.env.NODE_ENV !== 'production'
```

Comment:
Should we just name this `isProd` and inverse the condition? I'm worried we'll end up with usages of this that look like `if (!isNotProd)` which introduces a confusing double negative boolean, always good to ignore those in my experience :sweat_smile: 

```suggestion
export const isDev: boolean = process.env.NODE_ENV === 'development'
export const isTest: boolean = process.env.NODE_ENV === 'test'
export const isProd: boolean = process.env.NODE_ENV === 'production'
```

Of course we'll need to update where it's used as well.

Reply:
Thanks for noticing this, resolved in [ad3b2b1](https://github.com/WordPress/openverse-frontend/pull/1023/commits/ad3b2b18b38e1868715e792009bf0c33be0bcaab).